src/components/ErrorModal.vue

```diff
@@ -0,0 +1,140 @@
+<template>
+  <div
+    class="modal-mask"
+    @click="resolve"
+  >
+    <div class="modal-wrapper" @click.stop>
+      <div class="error-modal">
+        <h1>{{ $t('reportBug') }}</h1>
+        <textarea
+          v-model="report"
+          class="report-text"
+        />
+        <button
+          class="button"
+          @click="send"
+        >
+          {{ $t('sendBugReport') }}
+        </button>
+      </div>
+    </div>
+  </div>
+</template>
+
+<script>
+import { detect } from 'detect-browser';
+
+import SuccessIcon from '../assets/verifiedUrl.svg';
+import FailureIcon from '../assets/iconError.svg';
+import { backendFetch } from '../utils/backend';
+
+export default {
+  props: {
+    failure: { type: Boolean },
+    hideIcon: { type: Boolean },
+    resolve: { type: Function, required: true },
+    error: { type: [String, Number], default: 'Something went wrong' },
+  },
+  data() {
+    return {
+      SuccessIcon,
+      FailureIcon,
+      report: '',
+    };
+  },
+  methods: {
+    async send() {
+      const report = {
+        appVersion: process.env.npm_package_version,
+        browser: detect(),
+        error: this.error,
+        platform: 'web',
+        description: this.report,
+        time: new Date().toISOString(),
+      };
+
+      await backendFetch('errorreport', {
+        method: 'POST',
+        headers: {
+          'Content-Type': 'application/json',
+        },
+        body: JSON.stringify(report),
+      });
+    },
+  },
+};
+</script>
+
+<style lang="scss" scoped>
+.modal-mask {
+  vertical-align: middle;
+  padding-top: 5rem;
+  position: fixed;
+  z-index: 9998;
+  top: 0;
+  left: 0;
+  width: 100%;
+  height: 100%;
+  display: table;
+  background-color: rgba(0, 0, 0, 0.5);
+
+  .error-modal {
```

Comment:
Let's use flex instead of `display: table`

```suggestion
  display: flex;
  background-color: rgba(0, 0, 0, 0.5);

  .error-modal {
    margin: auto;
```